underfs/hdfs/src/test/java/alluxio/underfs/hdfs/HdfsUnderFileSystemTest.java

```diff
@@ -63,4 +87,85 @@ public void prepareConfiguration() throws Exception {
     Assert.assertEquals(ufsConf.get(PropertyKey.UNDERFS_HDFS_IMPL), conf.get("fs.hdfs.impl"));
     Assert.assertTrue(conf.getBoolean("fs.hdfs.impl.disable.cache", false));
   }
+
+  class PreadSeekableStream extends FilterInputStream implements Seekable, PositionedReadable {
```

Comment:
nit: this doesn't need to be a nested class? separate file for readability

Reply:
done